lib/modulesync/settings.rb

```diff
@@ -12,8 +12,17 @@ def initialize(global_defaults, defaults, module_defaults, module_configs, addit
       @additional_settings = additional_settings
     end
 
+    def lookup_config(hash, filename)
+      fn = filename.chomp('.erb')
+      hash[fn] || hash["#{fn}.erb"] || {}
```

Comment:
Assume we are talking about `rubocop.yml.erb` producing `.rubocop.yml`

In my imaginary world of best practices (which conforms to yours in this case) the settings files should key off of `.rubocop.yml.erb` and not `.rubocop.yml` this conditional return was to "just work" when someone keys off of .rubocop.yml (even though they are messing with .erb)... I am happy to implement what you suggested though if you don't see any value in permitting consumer sloppiness.